src/ESPeasyGPIO.ino

```diff
@@ -0,0 +1,396 @@
+/****************************************************************************/
+// Central functions for GPIO handling
+// **************************************************************************/
+#include "EspEasyGPIO.h"
+
+//********************************************************************************
+// Internal GPIO write
+//********************************************************************************
+void GPIO_Internal_Write(byte pin, byte value)
+{
+  pinMode(pin, OUTPUT);
+  digitalWrite(pin, value);
```

Comment:
Does not check using `checkValidPortRange`

Reply:
ok